src/containers/ArticleListing/index.js

```diff
@@ -21,13 +21,26 @@ export class ArticleListing extends React.Component {
     page: 1,
   };
 
+  constructor(props) {
+    super(props);
+
+    this.load = this.load.bind(this);
+  }
+
   componentDidMount = () => {
     const {
       url, fetchArticles, params, listName,
     } = this.props;
     fetchArticles(url, params, listName);
   };
 
+  load() {
+    const {
+      fetchArticles, url, params, listName,
+    } = this.props;
+    fetchArticles(url, params, listName);
```

Comment:
This line has not been covered yet

Reply:
The load function is not used anywhere, I forgot to remove it..so am removing it now...